src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java

```diff
@@ -255,23 +277,52 @@ public static Map<String, Object> getPersonalisationForAcknowledgement(CaseDetai
         );
     }
 
-    private static TseReplyData createDataForTseReply(CaseData caseData, GenericTseApplicationType application) {
+    private static TseDecisionData createDataForTseDecision(CaseData caseData) {
 
-        return TseReplyData.builder()
+        return TseDecisionData.builder()
             .caseNumber(defaultIfEmpty(caseData.getEthosCaseReference(), null))
-            .respondentParty(RESPONDENT_TITLE)
-            .type(defaultIfEmpty(application.getType(), null))
-            .responseDate(UtilHelper.formatCurrentDate(LocalDate.now()))
-            .response(defaultIfEmpty(application.getDetails(), null))
-            .supportingYesNo(hasSupportingDocs(caseData.getTseResponseSupportingMaterial()))
-            .documentCollection(getUploadedDocList(caseData))
-            .copy(defaultIfEmpty(application.getCopyToOtherPartyYesOrNo(), null))
+            .notificationTitle(caseData.getTseAdminEnterNotificationTitle())
+            .decision(caseData.getTseAdminDecision())
+            .decisionDetails(caseData.getTseAdminDecisionDetails())
+            .typeOfDecision(caseData.getTseAdminTypeOfDecision())
+            .responseRequired(YES.equals(caseData.getTseAdminIsResponseRequired())
+                    ? TSE_ADMIN_DECISION_RESPONSE_REQUIRED_MESSAGE : NO)
+            .selectedRespondents(caseData.getTseAdminSelectPartyRespond())
+            .additionalInformation(caseData.getTseAdminAdditionalInformation())
+            .supportingMaterial(getSupportedDocumentsList(caseData))
+            .decisionMadeBy(caseData.getTseAdminDecisionMadeBy())
+            .decisionMakerFullName(caseData.getTseAdminDecisionMadeByFullName())
+            .notificationParties(caseData.getTseAdminSelectPartyNotify())
             .build();
     }
 
+    private static List<DocumentType> getSupportedDocumentsList(CaseData caseData) {
+        List<GenericTypeItem<DocumentType>> documentList = new ArrayList<>();
+        if (!CollectionUtils.isEmpty(caseData.getTseAdminResponseRequiredYesDoc())) {
+            documentList.addAll(caseData.getTseAdminResponseRequiredYesDoc());
+        }
+        if (!CollectionUtils.isEmpty(caseData.getTseAdminResponseRequiredNoDoc())) {
+            documentList.addAll(caseData.getTseAdminResponseRequiredNoDoc());
+        }
+        List<DocumentType> documents = new ArrayList<>();
+        for (GenericTypeItem<DocumentType> document : documentList) {
+            DocumentType documentType = new DocumentType();
+            documentType.setShortDescription(document.getValue().getShortDescription());
```

Comment:
```suggestion
            documentType.setShortDescription(defaultIfEmpty(document.getValue().getShortDescription(), ""));
```

Reply:
Short description is an optional field so the value may or may not be present. Best to put a default value around it if is it null